Twig/Extension/ThemeExtension.php

```diff
@@ -44,33 +44,42 @@ public function getFunctions()
 
     /**
      * Displays Wordpress theme header (and administration menu bar if available).
+     *
+     * @param string|null $name
      */
-    public function getHeader()
+    public function getHeader($name = null)
     {
-        \get_header();
+        \get_header($name);
 
         \_wp_admin_bar_init();
         \wp_admin_bar_render();
     }
 
     /**
      * Displays Wordpress theme sidebar.
+     *
+     * @param string|null $name
      */
-    public function getSidebar()
+    public function getSidebar($name = null)
     {
-        \get_sidebar();
+        \get_sidebar($name);
     }
 
     /**
      * Displays Wordpress theme footer.
+     *
+     * @param string|null $name
      */
-    public function getFooter()
+    public function getFooter($name = null)
     {
-        \get_footer();
+        \get_footer($name);
     }
 
     /**
      * Displays a Wordpress theme template part.
+     *
+     * @param string $name
```

Comment:
It should be $slug here please ;)

Reply:
Wow! Sorry) This was a 2 a.m. when I write this)